ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -523,3 +548,51 @@ decodeShelleyResult query = case query of
     GetStakeDistribution         -> fromCBOR
     GetFilteredUTxO {}           -> fromCBOR
     GetUTxO                      -> fromCBOR
+    GetCurrentLedgerState        -> unwrapCBORinCBOR (const <$> decode)
+
+
+-- Following instances required for the Serialise instances for LedgerState.
+-- At some stage they should be pushed down into the ledger-specs repo.
+deriving instance Generic SL.Coin
```

Comment:
I would normally put orphan instances in a separate `Orphans` module, but that does not seem to the case in this repo.

Reply:
See my comment above

Reply:
Have defined:
```
fromCborInCbor :: FromCBOR a => Decoder s a

toCborInCbor :: ToCBOR a => a -> Encoding
```